### DISCUSSION TITLE
2021-10: 老年玩家只会种田

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#224] [Announcements]
https://xuanwo.io/reports/2021-10/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by STRRL** [comment]
有没有兴趣一起来 factorio 无虫群建设大规模自动化基地呀！

**Comment by Xuanwo** [comment]
> 有没有兴趣一起来 factorio 无虫群建设大规模自动化基地呀！

石油都不会开采也可以吗🥲🥲🥲

**Comment by STRRL** [comment]
俺也不会! 前期对着蓝图贴贴贴 后期有了 LTN 火车就起飞了 (指在 factorio 里搞 producer/consumer 🙈
